templates/aframe-model-template/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "aframe-model-template",
+  "description": "A WebVR scene. Made with A-Frame, a WebVR framework.",
+  "author": "A-Frame authors",
+  "version": "0.1.1",
+  "license": "CC0-1.0",
+  "repository": "aframevr-userland/aframe-default-template",
```

Comment:
we'll want to change this to `aframevr-userland/aframe-model-template`, and I can help you push these to https://github.com/aframevr-userland/ (I eventually want to move these all to the org, and not keep them in this project – I think; I'm still evaluating the workflow for creating new boilerplates)